plugins/catalog-backend/src/database/Database.test.ts

```diff
@@ -150,6 +157,45 @@ describe('Database', () => {
     });
   });
 
+  describe('locationHistory', () => {
+    it('outputs the history correctly', async () => {
+      const catalog = new Database(database, getVoidLogger());
+      const location: AddDatabaseLocation = { type: 'a', target: 'b' };
+      const { id: locationId } = await catalog.addLocation(location);
+
+      await catalog.addLocationUpdateLogEvent(
+        locationId,
+        DatabaseLocationUpdateLogStatus.SUCCESS,
+      );
+      await catalog.addLocationUpdateLogEvent(
+        locationId,
+        DatabaseLocationUpdateLogStatus.FAIL,
+        undefined,
+        'Something went wrong',
+      );
+
+      const result = await catalog.locationHistory(locationId);
+      expect(result).toEqual([
+        {
+          created_at: expect.anything(),
```

Comment:
Could be good to check types here too to be sure? `expect.any(Date)` or whatever it is :grin:

Reply:
I don't think the DB layer spits out the Date here for now, but we should definitely look into it